src/main/java/com/celements/web/plugin/cmd/PageLayoutCommand.java

```diff
@@ -377,16 +376,18 @@ public SpaceReference getPageLayoutForDoc(DocumentReference documentReference) {
    * @return
    */
   public boolean checkLayoutAccess(SpaceReference layoutSpaceRef) {
-    String layoutWikiName = layoutSpaceRef.getParent().getName();
-    return getContext().getDatabase().equals(layoutWikiName) || "celements2web".equals(
-        layoutWikiName);
+    if (layoutSpaceRef != null) {
+      String layoutWikiName = layoutSpaceRef.getParent().getName();
+      return getContext().getDatabase().equals(layoutWikiName) || "celements2web".equals(
+          layoutWikiName);
+    }
+    return false;
```

Comment:
First: This changes the contract of the method. Before the parameter layoutSpaceRef was NotNullable. 
Second: I am not convinced that "false" is a good default for LayoutAccess. This method should only be used for view-access and therefore a default-value of "true" seams to me more reasonable.